test/regression-c-semantics/.gitignore

```
@@ -0,0 +1 @@
+definition.kore
```

```suggestion
definition.kore

```
Don't we want a new line at the end here?